@here/olp-sdk-dataservice-read/lib/client/PartitionsRequest.ts

```diff
@@ -17,24 +17,45 @@
  * License-Filename: LICENSE
  */
 
+import { QuadKey } from "@here/olp-sdk-dataservice-read";
+
 /**
  * A class that prepare information for calls to get Partitions metadata from MetadataAPI.
  */
 export class PartitionsRequest {
-    private version: number | undefined;
+    private version?: number;
+    private quadKey?: QuadKey;
 
     public getVersion(): number | undefined {
         return this.version;
     }
 
+    public getQuadKey(): QuadKey | undefined {
```

Comment:
As far as i remeber we said we will separate the QuadKey partitions request from the one with only partitions IDs:

- getPartitions(QuadKeyPartitionsRequest) with QuadKeyPartitionsRequest().withQuadKey(quadKey).withVersion(version).withDepth(depth)

- getPartitions(PartitionsRequest) with PartitionsRequest().withVersion(version).withPartitionIds(list)